deps/rabbitmq_web_mqtt/src/rabbit_web_mqtt_handler.erl

```diff
@@ -156,7 +155,9 @@ websocket_info({'$gen_cast', duplicate_id}, State = #state{ proc_state = ProcSta
                                                             conn_name = ConnName }) ->
     ?LOG_WARNING("Web MQTT disconnecting a client with duplicate ID '~s' (~p)",
                  [rabbit_mqtt_processor:info(client_id, ProcState), ConnName]),
-    stop(State);
+    rabbit_mqtt_processor:send_disconnect(?RC_SESSION_TAKEN_OVER, ProcState),
+    defer_close(?CLOSE_PROTOCOL_ERROR),
```

Comment:
Should we rather use here `?CLOSE_NORMAL`?
A session being taken over is not really a WebSocket protocol error. It rather fits the description of the Status Code 1000 in the sense that the purpose of the WebSocket / MQTT connection has been fulfilled because a new network connection (but same MQTT session) with the same Client ID exists:
> 1000 indicates a normal closure, meaning that the purpose for which the connection was established has been fulfilled